spec/forms/api/v1/user/lessons/index_form_spec.rb

```diff
@@ -2,16 +2,26 @@
 
 RSpec.describe 'Api::V1::User::Lessons::IndexForm' do
   describe '#index' do
-    let(:index_form) { Api::V1::User::Lessons::IndexForm.new }
+    let(:index_form) { Api::V1::User::Lessons::IndexForm.new(params) }
 
-    context 'lessonが存在する場合' do
+    context 'pageがある場合' do
       let!(:lessons) { create_list(:lesson, 2) }
+      let(:params) { { page: 1 } }
 
       it '正しいレスポンスが返ってくること' do
         expect(index_form.index).to eq ApiResponse.base_response(
           Api::V1::User::LessonsResponse.index_success(lessons), nil, STATUS_SUCCESS
```

Comment:
例えば、こういうことじゃないん？（テスト通るか分からんけど）
```suggestion
          Api::V1::User::LessonsResponse.index_success(lessons.first(10)), nil, STATUS_SUCCESS
```
`lessons`には11個以上のテストデータが含まれる